src/test/java/com/flightstats/hub/metrics/MetricsConfigTest.java

```diff
@@ -0,0 +1,54 @@
+package com.flightstats.hub.metrics;
+
+import com.flightstats.hub.app.HubHost;
+import com.flightstats.hub.app.HubProperties;
+import com.flightstats.hub.app.HubVersion;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.testng.AssertJUnit.assertEquals;
+import static org.testng.AssertJUnit.assertFalse;
+
+
+
+public class MetricsConfigTest {
+    @Test
+    public void testDefaultValues_defaultTags() {
```

Comment:
With the other changes I ended up making, it made sense to change this to testing for static expectations after all.